src/native/src/3rdparty/json/ChangeLog.md

```diff
@@ -0,0 +1,224 @@
+# Change Log
```

Comment:
[Subtrees](http://blogs.atlassian.com/2013/05/alternatives-to-git-submodule-git-subtree/) look pretty good too. What do you guys think?